src/analysis/convex/basic.lean

```diff
@@ -423,6 +423,21 @@ begin
   exact h hx hy ha hb hab
 end
 
+lemma convex_iff_forall_pos_ne :
```

Comment:
Is there any point keeping `convex_iff_forall_pos` around? Either you need the strong forward direction, which is just `rw convex`, or you need the strong backwards direction, which is `rw <-convex_iff_forall_pos_ne`. I think having an intermediate strength version is unecessary.

Reply:
I think it's fine to keep? We can nuke it later if we want to.